test/org/traccar/reports/ReportUtilsTest.java

```diff
@@ -95,12 +95,20 @@ public void testDetectTripsSimple() throws ParseException {
         assertEquals(10, itemTrip.getMaxSpeed(), 0.01);
         assertEquals(3000, itemTrip.getDistance(), 0.01);
 
-        result = ReportUtils.detectTripsAndStops(tripsConfig, false, 0.01, data, false);
+        Collection<StopReport> stops = ReportUtils.detectStops(data, tripsConfig, false, 0.01);
 
-        assertNotNull(result);
-        assertFalse(result.isEmpty());
+        assertNotNull(stops);
+        assertFalse(stops.isEmpty());
 
-        StopReport itemStop = (StopReport) result.iterator().next();
+        Iterator<StopReport> iterator = stops.iterator();
+
+        StopReport itemStop = iterator.next();
+
+        assertEquals(date("2016-01-01 00:00:00.000"), itemStop.getStartTime());
```

Comment:
It is the first difference in behavior: initial parking/trip will detected even if it is too short.